modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -42,16 +43,27 @@ Some features in this release are currently in Technology Preview (TP). These ex
 |====
 |Feature |TP in OCP versions|GA in OCP versions
 
+|ApplicationSet Progressive Rollout Strategy
+|4.10, 4.11, 4.12
+|NA
+
+|Multiple sources for an application
+|4.10, 4.11, 4.12
+|NA
+
 |Argo CD applications in non-control plane namespaces
 |4.8, 4.9, 4.10, 4.11, 4.12
 |NA
 
+|Argo CD Notifications controller
+|4.8, 4.9, 4.10, 4.11, 4.12
```

Comment:
@harrietgrace Are we planning to support Argo CD Notifications controller and kam on OCP < 4.10?

Reply:
The notifications controller is part of Argo CD, so it will inherit the support of the version it was introduced with. We added notifications as TP in 1.6.0 which supports 4.8 through 4.11. Once 1.6.0 goes EOL, then the notifications TP will be supported on 1.7.0 and up, which inherits 4.10 through 4.12 support. 

Does that make sense? This table might require an adjustment to show the GitOps Operator version of each feature, rather than focusing on the OCP support.

Reply:
Okay, thanks @harrietgrace, makes sense

Reply:
I'll make a separate jira to change this table over to GitOps versions rather than OCP versions. I don't think it's urgent enough to delay the release or rush out the change.

Reply:
@harrietgrace As per our discussion the last day, I have made the following changes in the TP table:

- Removal of kam from TP table as it is a component and not a feature and is already captured in the matrix table
- Update to TP table with GitOps versions

PTAL  at the revised PR and approve.

Cc: @varshab1210

Reply:
setting this to resolved.